src/modules/strings.c

```diff
@@ -0,0 +1,193 @@
+#include "wren.h"
+
+int TclByteArrayMatch(
+    const char *string,
+    int strLen,
+    const char *pattern,
+    int ptnLen,
+    int unused
+);
+
+void stringsGlobMatch(WrenVM *vm) {
+    const char* string = wrenGetSlotString(vm, 1);
+    int strLen = (int)wrenGetSlotDouble(vm, 2);
+    const char* pattern = wrenGetSlotString(vm, 3);
+    int ptnLen = (int)wrenGetSlotDouble(vm, 4);
+    int result = TclByteArrayMatch(string, strLen, pattern, ptnLen, 0);
+    wrenSetSlotBool(vm, 0, result);
+}
+
+/*
+ * tclUtil.c --
```

Comment:
I think this (just the TCL stuff) probably should go in `src/vendor` (like in the JSON PR), but if you're tired of fooling around here I can jump in and polish it up.  Let me know.

This is looking super awesome.